### PR TITLE
Preprocessor to correct header/text mistakes

### PR DESCRIPTION
**Pull request by tadhg-ohiggins**
Fixes https://github.com/18F/fec-eregs/issues/6

### CODE REVIEW

File: eregs_extensions/fec_regparser/preprocs/__init__.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        <P>(d) <E T="03">[Term]. [Term]</E> [text]</P>
+
+    This preprocessor tries to correct them appropriately.
+    """
+
+    def transform(self, xml):
+        for el in xml.xpath("//P//E[contains(., '.')]"):
+            pair = [_.strip() for _ in el.text.split(".")]
+
+            if len(pair) == 2 and pair[0] == pair[1]:
```
Comment (cmc333333) [inline eregs_extensions/fec_regparser/preprocs/__init__.py]: This is a pretty solid guard

**Comment by cmc333333** [comment]
Logic makes sense to me, though I'd recommend adding tests

**Comment by tadhg-ohiggins** [comment]
Tests added.

**Comment by adborden** [comment]
:+1:

**Pull request merged**